darts/models/smoothing_model.py

```diff
@@ -0,0 +1,109 @@
+"""
+Smoothing Model Base Class
+------------------------------
+
+A smoothing model more accurate value from measured values of a time series
+Using the current value and historic values as follows:
+
+.. math:: y_{t+1} = f(y_{t+1}, y_t, y_{t-1}, ..., y_1),
```

Comment:
This would mean that these models always represent current values as a function of the past only. I think that's not accurate (e.g. moving average at position t+1 also depends of signal at position t+2). This equation was useful for forecasting models but perhaps not really needed for filtering models...
If you want to keep it perhaps rewrite it as something like \hat{y}_{t} = f(y_1, ..., y_t, ... y_T).

Reply:
is this valid? its past and current value?